docs/design/2018-10-24-view-support.md

```diff
@@ -5,65 +5,119 @@
 - Discussion at: https://github.com/pingcap/tidb/issues/7974
 
 ## Abstract
-This proposal proposes to implement VIEW feature in TiDB, aimed to make SQL easier to write.
+This proposal proposes to implement basic VIEW feature in TiDB, aimed to make SQL easier to write. VIEW's advanced feature would be considered later.
 
 ## Background
 A database view is a searchable object in a database that is defined by a query. Though a view doesn’t store data, some refer to a VIEW as "virtual tables", and you can query a view like you can query a table. A view can combine data from two or more tables, using joins, and also just contain a subset of information. This makes them convenient to abstract, or hide, complicated queries.
 
 Below is a visual depiction of a view:
-  ![AnatomyOfAview](imgs/view.png)
+  ![AnatomyOfAview](imgs/view.png)   
+  reference from https://www.essentialsql.com/what-is-a-relational-database-view/
   
 A view is created from a query using the "`CREATE OR REPLACE VIEW`" command. In the example below we are creating a PopularBooks view based on a query which selects all Books that have the IsPopular field checked. Following is the query:
-`CREATE OR REPLACE VIEW PopularBooks AS SELECT ISBN, Title, Author,PublishDate FROM Books WHERE IsPopular=1`  
+```mysql
+CREATE OR REPLACE 
+  VIEW PopularBooks AS 
+    SELECT ISBN, Title, Author,PublishDate FROM Books WHERE IsPopular=1
+```  
 
 Once a view is created, you can use it as you query any table in a `SELECT` statement. For example, to list all the popular book titles ordered by an author, you could write:  
-`SELECT Author, Title FROM PopularBooks ORDER BY Author`  
+```mysql
+SELECT Author, Title FROM PopularBooks ORDER BY Author
+```
 
 In general you can use any of the SELECT clauses, such as GROUP BY, in a select statement containing a view.
 
 ## Proposal
-I implement the "`CREATE OR REPLACE`" command to create a view with the `SELECT` statement. Currently, I reuse "`DROP TABLE`" to drop a view, and "`SHOW FULL TABLES`" can also display the view list with different table types. The "`SHOW CREATE TABlE`" command can also regenerate a view create command.
-
+This proposal is prepared to implement the basic VIEW feature, which contains `CREATE OR REPLACE VIEW`, `SELECT FROM VIEW`, `DROP VIEW` and `SHOW TABLE STATUS`. All other unimplemented features will be listed for compatibility and discussed later.
```

Comment:
This proposal is prepared to implement the basic VIEW feature, which contains:
- `CREATE OR REPLACE VIEW`
- `SELECT FROM VIEW`
- `DROP VIEW`
- `SHOW TABLE STATUS`